ext/node/ops/v8.rs

```diff
@@ -30,3 +39,343 @@ pub fn op_v8_get_heap_statistics(
   buffer[12] = stats.used_global_handles_size() as f64;
   buffer[13] = stats.external_memory() as f64;
 }
+
+pub struct Serializer<'a> {
+  inner: v8::ValueSerializer<'a>,
+}
+
+pub struct SerializerDelegate {
+  obj: v8::Global<v8::Object>,
```

Comment:
could this be a `v8::TracedReference`?

Reply:
Maybe? I'm just not sure how to actually call `trace` on it. The delegate is owned by `v8::ValueSerializer`, which stores it as a `Box<dyn ValueSerializerImpl>` and doesn't know that it's garbage collected.

I guess I could maybe keep a `Rc<SerializerDelegate>` (or equivalent) around in `Serializer`, and trace that?

Reply:
oh i see i misread this since the gc impl came after SerializerDelegate, but is written for Serializer.